### ISSUE TITLE
Invisible items in inventory

### ISSUE THREAD

**Issue by skellitor301** [BuildCraft/BuildCraft#4502]
- Forge Version: 14.23.5.2847
 - BuildCraft Version (Add Hash if from github): buildcraft-all-799.24.3.jar and buildcraft-all-799.24.2.jar
 - Any additonal mods:
/dank/null
AE2 Stuff
Agricraft
Animated Crosshair
Antique Atlas
Appleskin
Applied Energistics 2
Aroma1997Core
Aroma1997s Dimensional World
AromaBackup
Atlas Extras
AutoRegLib
Avarita 1.1x
Baubles
BaublesHud
Baubly Heart Canisters
BdLib
Bed Patch
Better Advancements
Better Animations Collection 2
BetterBedrockGen
BetterFps
BetterPortals
BiblioCraft
Biomes O'Plenty
Bonsai Trees
Brandon's Core
Chameleon
Chisels & Bits
CodeChicken Lib 1.8.+
CoFH Core
CoFH World
Comforts
Construct's Armory
Controlling
Cooking for Blockheads
Crafting Tweaks
Creeper Confetti
Cucumber Library
DIng
Doggy Talents
Doomlike Dungeons
Draconic Evolution
EleCore
Ender IO
Ender Storage 1.8.+
EnderCore
EnderTanks
Extra Bit Manipulation
Extra Utilities
ExtraCells2
Farseek
Fas Leaf Decay
Fence Overhaul
FoamFix for Minecraft
Forestry
Forge MultiPart CBE
Fullscreen Windowed (Borderless) For Minecraft
Hopper Ducts
iChunUtil
InfinityLib
Inventory Tweaks
Iron Chests
JourneyMap
Just Enough Energistics (JEE)
Just Enough Items (JEI)
Just Enough Resources (JER)
KleeSlabs
LibVulpes
LunatriusCore
MalisisCore
MalisisDoors
Mantle
Mercurius
Minecolonies
More Player Models
Mouse Tweaks
MrTJPCore
Multi Mine
Mystic World
Mystical Adaptations
Mystical Agradditions
Mystical Agriculture
MysticalLib
Neat
OMLib
Open Modular Turrets
P455w0rd's Library
Pam's HarvestCraft
Patchouli
Project Intelligence
Project Red - Base
Project Red - Compat
Project Red - Fabrication
Project Red - Integration
Project Red - Lighting
Project Red - Mechanical
Project Red - World
Proportional Destruction Particles
Quark
Redstone Flux
RPG-Hud
Shadowfacts' Forgelin
ShetiPhianCore
Signpost
Snad
StepUp
stimmedcow : NoMoreRecipieConflic
Storage Drawers
Storage Drawers Extras
Streams
SwingThroughGrass
Sync
The One Probe
Thermal Dynamics
Thermal Expansion
Thermal Foundation
Thermal Innovation
Thermal Tinkering
Tinker I/O
Tinker's JEI
Tinkers Construct
Tinkers Mechworks Fork
Tinkers' Compendium (Tinkers' Defense)
Tinkers' Complement
Tiny Progressions
Torch Optimizer
Translocators 18.+
UniDict
VeinMiner
ViesCraft - Airships!
Waddles
WanionLib
Waystones

Description:
When I open JEI to look up items for buildcraft some of the items like the engines, some wires, etc are invisible in JEI and in the inventory. When I place the item it appears like normal, but when it's in it's item form (I.E. dropped on the ground or in the inventory) the item is not rendered at all.
![javaw_2019-10-21_15-22-01](https://user-images.githubusercontent.com/3630718/67238327-f9df5180-f41a-11e9-98b7-1790a5b035b6.png)
![javaw_2019-10-21_15-22-09](https://user-images.githubusercontent.com/3630718/67238328-f9df5180-f41a-11e9-93ab-571553b87088.png)

**Comment by AlexIIL** [comment]
I suspect this is a compatibility problem with one of your mods that touches models - can you reproduce with a much smaller mod list? (Ideally just buildcraft + the mod(s) that causes it).

**Comment by skellitor301** [comment]
I wouldn't know which mods would touch models like that, I can provide an exported modpack from the curse launcher if that would help?

**Comment by AlexIIL** [comment]
Not really, as it might be specific to how you've setup your mods. Generally the best way to find out which mod causes this is to remove half of them, then try again. (And keep removing half of what you've got until it starts working again, then you can add half of the ones you removed etc until you've found out the one that cause it (Or remove all except buildcraft to see if it happens with *just* buildcraft + forge).

> Would there be a list of mods that are known to be compatible with BC so the list can be whittled down?

Generally any "content" mods (like Iron Chests, Thermal *, or tinkers) probably don't cause this, however optimisation mods (like foamfix, or BetterFps) might.

**Comment by JoshPJackson** [comment]
Could this be related to https://github.com/BuildCraft/BuildCraft/issues/4504?

**Comment by AlexIIL** [comment]
I wouldn't have thought so as skellitor301's modlist doesn't include optifine.

**Comment by skellitor301** [comment]
Actually I think I do have optifine in that modpack cause I remember shaders being used.

I just checked and yes, optifine is in there specifically the 1.12.2 HD_U_E3 version

**Comment by AEnterprise** [comment]
try turning off anti-aliassing, if that fixes it then this is indeed #4504

**Comment by Nesquilk** [comment]
Por favor me responda, sou usuário de Minecraft bedrock e após aceitar todas as opções na criação de mundo (por parte de mecânica), estou tendo este mesmo problema de invisibilidade com itens (por parte de inventário ou drop), por favor me ajude

**Comment by Nesquilk** [comment]
@AlexIIL poderia me ajudar, por favor?
